guide/getting-started-nodejs.md

```diff
@@ -0,0 +1,139 @@
+## Getting started with NodeJS on OpenFaaS
+
+There are a number of resources that document getting started with
+OpenFaaS, in fact the entire stack can be deployed in less than 60
+seconds when following the instructions from the README on GitHub!
+This article will focus on how easy it is to setup a simple NodeJS
```

Comment:
I think we could prefer a plain-language style:

> This guide walks you through creating your first Node.js function.

Reply:
sounds good. I'll make some the changes

Reply:
Did this change go in?

Reply:
@alexellis Yes, i cut the paragraph and added the exact line above, it might be helpful to remove this pull request and resubmit?

Reply:
I'm still seeing the word "article" here though? The PR doesn't need removing, are you sure it's synced up?